### DISCUSSION TITLE
Annotate a multiarch manifest list

### DISCUSSION THREAD

**Discussion by emarsden** [containers/podman#21812] [Q&A]
I am trying to add annotations to a multiarch manifest list, as [described here](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#adding-a-description-to-multi-arch-images) for the ghcr. 

When I use the digest from "podman manifest create" as an argument to "podman manifest annotate", podman returns an error ""invalid image digest [...] invalid checksum digest format". It seems to be trying to annotate the image manifests listed in the multiarch manifest list, rather than provide an annotation at the top level of the manifest list. This is with podman version 4.9.3 on Linux.

I see also that the mediaType for the manifest list is "application/vnd.docker.distribution.manifest.list.v2+json", but the [Opencontainers specification](https://github.com/opencontainers/image-spec/blob/main/media-types.md#compatibility-matrix) seems to indicate that only OCI manifest lists (application/vnd.oci.image.manifest.v1+json) allow annotations.

**Comment by SpiGAndromeda** [comment]
A wonder about the same thing. Especially since the package view explicitly suggests that one should attach annotations to the manifest.
